src/NodaTime.Test/Xml/XmlSchemaTest.XmlSchema.approved.xml

```diff
@@ -110,6 +110,9 @@
     </xs:restriction>
   </xs:simpleType>
   <xs:simpleType name="zone">
+    <xs:union memberTypes="nodatime:zoneIds xs:string" />
+  </xs:simpleType>
+  <xs:simpleType name="zoneIds">
```

Comment:
It's just occurred to me that changes in the set of zone IDs is presumably going to mean this file needs to be rewritten. I'd want to automate that. Please could you provide instructions for how you generated this file, so I can add that to the time zone update script?

Reply:
Here's how to generate this file:
```csharp
var settings = new XmlWriterSettings { Indent = true, NewLineChars = "\n", Encoding = new UTF8Encoding() };
using var xmlWriter = XmlWriter.Create(@"src\NodaTime.Test\Xml\XmlSchemaTest.XmlSchema.approved.xml", settings);
NodaTimeXmlSchema.Write(xmlWriter);
```
(Adapted from [NodaTime.Test.Xml.XmlSchemaTest.GetXmlSchemaString](https://github.com/nodatime/nodatime/blob/92e81ea52bfdb8088524515a3a0f37e7c274209c/src/NodaTime.Test/Xml/XmlSchemaTest.cs#L301-L309))

Also, it might be nice to host it on https://nodatime.org/api/NodaTime.xsd for example.

Reply:
Thanks. I've created a new issue to track the need for that.

Reply:
(The regeneration, at least. I'm not entirely convinced about the hosting yet, especially given that it depends on the time zone data in use.)